index.js

```diff
@@ -249,8 +249,8 @@ RingPop.prototype.bootstrap = function bootstrap(opts, callback) {
         return;
     }
 
-    this.checkForMissingBootstrapHost();
-    this.checkForHostnameIpMismatch();
+    checkForMissingBootstrapHost();
```

Comment:
This is incidental cleanup. I very much disliked how these were part of the prototype.